rosidl_generator_dds_idl/cmake/rosidl_generate_dds_interfaces.cmake

```diff
@@ -30,64 +31,50 @@
 # @public
 #
 macro(rosidl_generate_dds_interfaces target)
+  set(SERVICE_TEMPLATES OPTIONAL)
   cmake_parse_arguments(_ARG "" "EXTENSION"
-    "IDL_FILES;DEPENDENCY_PACKAGE_NAMES;OUTPUT_SUBFOLDERS" ${ARGN})
+    "IDL_TUPLES;DEPENDENCY_PACKAGE_NAMES;OUTPUT_SUBFOLDERS;SERVICE_TEMPLATES" ${ARGN})
```

Comment:
It would be nice to document `SERVICE_TEMPLATES` with the other parameters above.

Reply:
Added in 7fb85964fd5ecff6f6972067f2a27cd7b6fb39f7.